roles/openshift_health_checker/library/etcdstats.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+Ansible module to determine the size of an etcd cluster
+'''
+
+import etcd
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def get_size(client, key, seen=None):
```

Comment:
Had this conversation in person but wanted to post it here for broader audience ... asking the database to recursively list all keys will not only generate a huge amount of IO but also blow out the caches it's got set up, not to mention that asking to do a full recursive list on a production etcd close to the limit may cripple it. 

I'm not sure this is a super useful check to be doing with a client like this. If the etcd project has a maximum safe storage size, they should expose a server endpoint for monitoring this.